generators/workspace/templates/plain_initialize_once/config/webpack.dev.js

```diff
@@ -272,6 +273,14 @@ const config = {
             'process.env.__APP_CONTEXT__': JSON.stringify('/'),
             'process.env.__DEBUG__': JSON.stringify(isDev)
         }),
+        new Dotenv({
+            path: base + '/.env', // load this now instead of the ones in '.env'
+            safe: false, // load '.env.example' to verify the '.env' variables are all set. Can also be a string to a different file.
+            allowEmptyValues: true, // allow empty variables (e.g. `FOO=`) (treat it as empty string, rather than missing)
+            systemvars: true, // load all the predefined 'process.env' variables which will trump anything local per dotenv specs.
+            silent: true, // hide any errors
```

Comment:
Do we really want to hide errors?

```suggestion
            silent: false, // hide any errors
```

Reply:
should be true; otherwise you get an error if you don't have a file

Reply:
Ok, this makes sense.